src/traces/scatterpolargl/index.js

```diff
@@ -106,6 +106,12 @@ function plot(container, subplot, cdata) {
         if(options.line && !scene.line2d) scene.line2d = true;
         if((options.errorX || options.errorY) && !scene.error2d) scene.error2d = true;
 
+        stash.tree = cluster(positions);
+
+        if(options.marker) {
```

Comment:
this should probably be `options.marker && count >= TOO_MANY_POINTS` like in `scattergl/convert.js`